ts/nni_manager/rest_server/restHandler.ts

```diff
@@ -160,7 +160,7 @@ class NNIRestHandler {
     }
 
     private startExperiment(router: Router): void {
-        router.post('/experiment', expressJoi(ValidationSchemas.STARTEXPERIMENT), (req: Request, res: Response) => {
+        router.post('/experiment', (req: Request, res: Response) => {
```

Comment:
we do not need to validate now?

Reply:
I think validate in launcher is enough.
It's not DRY to validate twice.